transcom/mymove
# INT B-21480 Full PPM Counseling Type

## [Agility ticket](tbd)
B-21480 https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21480

## Summary
All full PPM moves (meaning orders with only PPM shipments on them) will not be routed to the TOO and will only be routed to the service counselor.

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1.  Login as a customer or Service Counselor.
2. Create a PPM that is the only move on the orders, or any other moves on those orders are also PPM moves (with the following details)
3. Create the move with an origin location of Mayport, FL 32228 (NS Mayport FL) and a destination zip of 62225 (Scott Air Force Base) -Mayport, FL is the only gbloc that currently routes to the TOO for Prime Counseling (we are verifying a PPM does not route to Prime Counseling)
4. Verify the move is created, shows on the counselor queue, and is put in the status "Needs Service Counseling"
5. Login as TOO
6. Verify the TOO does not have the move in their queue 
7. search for the move
8. Verify the TOO does not have the MTO Service Items with the option to add "Counseling"

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![image](https://github.com/user-attachments/assets/83aac25e-7d62-4d21-9ca0-1511171b6aa5)
![image](https://github.com/user-attachments/assets/01f52a7e-842e-4b42-a3d1-e61256ae4724)
![image](https://github.com/user-attachments/assets/8b613f06-df21-4289-8360-caf3928c2914)
